docs/security-v2/03-04-ingress-egress-traffic.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Ingress and Egress traffic
+type: Details
+---
+
+## Ingress
+Kyma uses the [Istio Ingress Gateway](https://istio.io/latest/docs/reference/config/networking/gateway/) in order to handle all incoming traffic, manage TLS termination or handle mTLS communication between the cluster and external services. By default a `kyma-gateway` is configured as the main point of entry, which exposes all applications using the supplied domain and certificate.
```

Comment:
Don't now yet if here or in different section but we should also describe apiserver-proxy (as long as we have it) cause it's additional LB and additional cert (in most of the cases)